### ISSUE TITLE
Empty string if missing index in lang files

### ISSUE THREAD

**Issue by manupap1** [ZoneMinder/ZoneMinder#819]
No string is displayed in web pages if the requested index is missing in SLANG array.
I think the index should be displayed as a fallback method.
This simple function could be used in `web/includes/lang.php`:
```php
function translate( $name )
{
    global $SLANG;
    if ( array_key_exists( $name, $SLANG ) )
        return $SLANG[$name];
    else
        return $name;
}
```
This script could update all files in a row: 
```bash
#!/bin/bash
FPATH="web/ajax web/skins"
for folder in $FPATH
do
    find $folder -type f -name "*.php" -exec sed -i "s/\$SLANG\['\([a-zA-Z0-9\]*\)']/translate\('\1'\)/g" {} \;
done
```

**Comment by knnniggett** [comment]
Please test #833

**Comment by SteveGilvarry** [comment]
Closed via #833 and #845
